Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/placeholders/placeholderExecutionErrors.dfy.expect

```diff
@@ -0,0 +1,4 @@
+
+Dafny program verifier finished with 3 verified, 0 errors
+placeholderExecutionErrors.dfy(14,7): Error: A placeholder module may only be replaced once. Other replacing module: placeholderExecutionErrors.dfy(20,7)
+placeholderExecutionErrors.dfy(4,19): Error: Placeholder module must be replaced somewhere in the program
```

Comment:
I'd be a bit more specific that this only applies when compiling, just so the error makes more sense in isolation